src/ios/CDVSplashScreen.m

```diff
@@ -175,7 +181,8 @@ - (CDV_iOSDevice) getCurrentDevice
     device.iPhone6 = (device.iPhone && limit == 667.0);
     device.iPhone6Plus = (device.iPhone && limit == 736.0);
     device.iPhoneX  = (device.iPhone && limit == 812.0);
-
+    device.iPhoneXSMax  = (device.iPhone && limitNativ == 2688.0);
```

Comment:
typo? 
should probably be `limitNative` or `limit`?
(same in line below)

Reply:
this is also the reason why tests are failing on Travis for iOS

Reply:
Typo, limitNative